CHANGELOG.md

```diff
@@ -347,7 +348,7 @@
 # 14.0.0 Release notes
 
 ### Enhancements
-* Property keypath in RQL can be substituted with value given as argument. Use '$P<i>' in query string. (Issue [#7033](https://github.com/realm/realm-core/issues/7033))
+* Property keypath in RQL can be substituted with value given as argument. Use '$K\<i\>' in query string. (Issue [#7033](https://github.com/realm/realm-core/issues/7033))
```

Comment:
escaping the `<>` so it's not interpreted as an html tag. also fairly certain that the `P` here is supposed to be a `K` given my read of the PR, but please double check me.